users/views.py

```diff
@@ -103,6 +107,12 @@ def put(self, request):
         if dob:
             profile_.update(dob=dob)
 
+        if profile_img:
+            user = CustomUser.objects.get(id=request.user.id)
+            file = ProfileSerializer(user, data={'profile_img': profile_img}, partial=True)
```

Comment:
I did observe where the uploaded image is associated with the user. Please attach a screenshot demonstrating the profile-image as part of the User.

Reply:
The screenshot has been uploaded